.github/workflows/all.yml

```diff
@@ -204,21 +208,45 @@ jobs:
             docker run $DOCKER_IMAGE_ID scale --help
             docker run $DOCKER_IMAGE_ID pause --help
             docker run $DOCKER_IMAGE_ID resume --help
-      - name: Publish
-        if: github.event_name != 'pull_request'
+      - name: Publish Docker CI
         run: |
-          echo "REF=${{ github.ref }}"
+          VERSION_CI="${VERSION_CI#v}"
           echo "DOCKER_IMAGE_ID=$DOCKER_IMAGE_ID"
-          # Log into registry
-          echo "${{ secrets.DOCKER_PASS }}" | docker login -u "${{ secrets.DOCKER_USER }}" --password-stdin
+          echo "VERSION_CI=$VERSION_CI"
+          # Log into GHCR
+          echo "${{ secrets.CR_PAT }}" | docker login https://ghcr.io -u ${{ github.actor }} --password-stdin
+          docker tag "$DOCKER_IMAGE_ID" "ghcr.io/$DOCKER_IMAGE_ID/ci:$VERSION_CI"
+          docker push "ghcr.io/$DOCKER_IMAGE_ID/ci:$VERSION_CI"
```

Comment:
In this step image is published separated from "official" images and with VERSION_CI created in configure step

Reply:
I like using the `/ci` suffix for this, but do we need to push the image if it's not published below? I can see it being useful for testing, but not sure if we need this. @na-- @MStoykov WYT?

Reply:
Yeah, I don't think we should push _all_ images to github's container registry :confused: I don't see a benefit, it's just going to add noise and take up space...

While it's nice that we are going to build the images for every PR and run a brief sanity check, we should only push the actual images for `master` builds and version tags.

Reply:
Now for each build triggered by:

```
on:
  # Enable manually triggering this workflow via the API or web UI
  workflow_dispatch:
  push:
    branches:
      - master
    tags:
      - v*
  pull_request:
```

 it's pushed on GHCR `/ci`

Reply:
@salem84 Thanks for the recent changes, but it doesn't look like this issue was addressed. That is, we want to avoid pushing to GHCR on every PR, so this workflow should be triggered only on `master` commits or version tags.

I think just keeping the `if: github.event_name != 'pull_request'` condition on the "Publish Docker CI" step would do it, but maybe @na-- has a better suggestion.